<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Bảng điều khiển</string>
    </property>
    <widget class="QPushButton" name="btnEnableMotor">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>enable motor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDicrectMotion">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>dicrect motion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAddress">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnectServer">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>connect server</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblFindObject">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>find object</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPose">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vi tri:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>391</width>
      <height>371</height>
     </rect>
    </property>
    <property name="text">
     <string>Anh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTrue">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>490</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Đúng đối tượng</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFalse">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>490</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Sai đối tượng</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
